{
  "_annotation": {
    "cve_id": "CVE-2024-29190",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-29190.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:opensecurity:mobile_security_framework:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "6AFCDC53-2EB8-5B9C-8241-02252BE6EC1E",
                "versionEndIncluding": "3.9.5-beta",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}